fastlane_core/spec/itunes_transporter_spec.rb

```diff
@@ -13,22 +13,36 @@
       allow(SecureRandom).to receive(:uuid).and_return(random_uuid)
     end
 
-    def shell_upload_command(provider_short_name: nil, transporter: nil, jwt: nil, use_asset_path: false)
+    def shell_upload_command(provider_short_name: nil, transporter: nil, jwt: nil, use_asset_path: false, username: email, input_pass: password, api_key: nil)
       upload_part = use_asset_path ? "-assetFile /tmp/#{random_uuid}.ipa" : "-f /tmp/my.app.id.itmsp"
 
-      escaped_password = password.shellescape
-      unless FastlaneCore::Helper.windows?
-        escaped_password = escaped_password.gsub("\\'") do
-          "'\"\\'\"'"
+      username = username if username != email
+      escaped_password = input_pass != password ? input_pass : password
```

Comment:
I'm not sure I understand this line… wouldn't the whole `input_pass != password ? input_pass : password` just be equivalent to use `input_pass` directly? i.e. I think this code is equivalent to just `escaped_password = input_pass`, right?